lib/api/src/modules/settings.ts

```diff
@@ -0,0 +1,41 @@
+import { ModuleFn } from '../index';
+
+export interface SubAPI {
+  changeSettingsTab: (tab: string) => void;
+  closeSettings: () => void;
+  navigateToSettingsPage: (path: string) => Promise<void>;
+}
+
+export const init: ModuleFn = ({ store, navigate, fullAPI }) => {
+  const api: SubAPI = {
+    closeSettings: () => {
+      const {
+        settings: { lastTrackedStoryId },
+      } = store.getState();
+
+      if (lastTrackedStoryId) {
+        fullAPI.selectStory(lastTrackedStoryId);
+      } else {
+        navigate('/story/*');
```

Comment:
Does this have any effect? I didn't think that actually led to the first story being selected any more..

Reply:
I'm not sure what the best way to do it otherwise is 🤔

Reply:
It has the effect of closing the settings page and showing this screen:

![Screen Shot 2020-06-30 at 11 26 49 AM](https://user-images.githubusercontent.com/3035355/86157440-a1da0e00-bac4-11ea-9aa7-5c5f841b94aa.png)

I would have liked to navigate to the first story & spent some time trying to figure out how to do that but it seemed way more complicated than this. But, if anyone knows how to do that, I can change it up.

Reply:
We could add a `selectFirstStory` method to the stories module. 

How do you get into this state?

Reply:
You get into this state by triggering the release notes to open immediately when you start Storybook. When you do that, you don't have any story history & therefore there is no story to navigate "back" to.

Reply:
`selectFirstStory` seems reasonable -- the thing I was struggling with was getting the state of the stories in order. There was a `storiesHash` but it wasn't sorted. I'm looking now at `jumpToStory` and think I can copy that logic to sort & find the first story.

Reply:
Updated & got `selectFirstStory` to work 👌